test_scripts/WebEngine/WebEngineProjection/010_OnSCU_after_Show_widget.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Check that SDL transfers `OnSystemCapabilitiesUpdated` notification from HMI to the WEB_VIEW App
+-- if App sent `Show` RPC with `templateConfiguration` parameter for main window
```

Comment:
```suggestion
-- if App sent `Show` RPC with `templateConfiguration` parameter for widget window
```

Reply:
@VjKlepikov Done in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/31/commits/0edf462457338d9f507c51c6bd8bbb31ff284b6c#diff-548f6617f5f2e588456e93d6edb49cd5R5